typing/shape.ml

```diff
@@ -460,6 +460,12 @@ let of_path ~find_shape ~namespace =
     | Pident id -> find_shape ns id
     | Pdot (path, name) -> proj (aux Module path) (name, ns)
     | Papply (p1, p2) -> app (aux Module p1) ~arg:(aux Module p2)
+    | Pextra_ty (path, extra) -> begin
+        match extra with
+          Pcstr_ty _ -> aux Type path
+        | Pext_ty -> aux Extension_constructor path
+        | Pcls_ty -> aux Class_type path
+      end
```

Comment:
We believe that `Shape.of_path` is only used on module paths anyway, so this definition cannot be wrong. It is also probably correct.